packages/front-end/pages/_app.tsx

```diff
@@ -66,7 +66,11 @@ function App({
 
   useEffect(() => {
     // Load feature definitions JSON from GrowthBook API
-    fetch("https://cdn.growthbook.io/api/features/key_prod_cb40dfcb0eb98e44")
+    fetch(
+      process.env.NODE_ENV === "production"
+        ? "https://cdn.growthbook.io/api/features/key_prod_cb40dfcb0eb98e44"
+        : "https://cdn.growthbook.io/api/features/key_dev_676ef35b3e2f8f3f"
+    )
```

Comment:
I could use a sign-off on this ... Is there any reason we wouldn't want to use the dev endpoint for our Growthbook instance in development?

Reply:
cc @jdorn

Reply:
Looks good